ui/app/components/job-status/panel/deploying.hbs

```diff
@@ -90,12 +110,14 @@
                 <span class="alloc-health-indicator">
                   {{#if (eq health "healthy")}}
                     <FlightIcon @name="check" @color="#25ba81" />
+                  {{else if (eq health "unhealthy")}}
+                    <FlightIcon @name="x" @color="black" />
                   {{else}}
                     <FlightIcon @name="running" @color="black" class="not-animated" />
                   {{/if}}
                 </span>
               </span>
-              {{count}} {{capitalize health}}
+              {{count}} {{humanize health}}
```

Comment:
`humanize` to drop the `_` in `health_unknown`.

(in case you were wondering: we don't use `unknown` here because that has a dedicated orange colour associated on elements from "Node status unknown" / max client disconnect stuff.)